zproject/urls.py

```diff
@@ -287,6 +288,11 @@
     rest_path('realm/filters/<int:filter_id>',
               DELETE=delete_filter),
 
+    # realm/playgrounds -> zerver.views.realm_playgrounds
+    rest_path('realm/playgrounds',
+              POST=(add_realm_playground,
+                    {'intentionally_undocumented'})),
```

Comment:
Rather than tagging this as intentionally_undocumented, add it to the exclude list in `test_openapi.py`; we do intend to document endpoints like this one.